source/adminguide/systemvm.rst

```diff
@@ -552,3 +552,25 @@ The administrator can log in to the secondary storage VM if needed.
    :alt: Updating Console Proxy SSL Certificate
 .. |vr-upgrade.png| image:: /_static/images/vr-upgrade.png
    :alt: Button to upgrade VR to use the new template.
+
+Troubleshoot networks from System VMs
+-------------------------------------
+
+
+For troubleshooting of network issues in CloudStack hosted networks, CloudStack allows
+the administrator to execute network-utility commands (ping, traceroute or arping)
+remotely on system VMs.
+
+To run either a ping, traceroute or arping through the CloudStack UI:
+
+#. As an administrator, log in to the CloudStack UI.
+
+#. Navigate to Infrastructure > System VMs or Virtual Routers.
+
+#. Click on the Run Diagnostics button. |run-diagnostics-icon.png|
+
+#. A form will pop up similar to this;
+      |diagnostics-form.png|
```

Comment:
have you tried building this and checking that the images are output? I can't *see* anything that links your images placeholders to the actual images.  just needs checking, it may get picked up from being in default image dir.

Reply:
Added links to the images and check by building, works fine now.